### DISCUSSION TITLE
Feature request: jtop

### DISCUSSION THREAD

**Discussion by hannes09** [OE4T/meta-tegra#706] [Ideas]
Hi,

it would be great to have something linke this:

[https://github.com/rbonghi/jetson_stats](https://github.com/rbonghi/jetson_stats)

**Comment by ichergui** [comment]
Hey @hannes09 

For your information, Nvidia provides tools like  `tegrastats` that you can use.

[Jtop](https://github.com/rbonghi/jetson_stats) have some python dependencies and it could be work fine with Nvidia Linux `ubuntu`, I don't think that it could be integrate in `meta-tegra` because it is dedicated for the `BSP`, maybe in [tegra-demo-distro](https://github.com/rbonghi/jetson_stats)

**Comment by dat-tran-matternet** [comment]
I haven't tried tegrastats yet but tried to create a recipe for python3-jetson-stats.bb but it seems to install systemd services to the host computer instead of the target. I'll try tegrastats but if anyone have a working jetson-stas recipe please share. thanks

**Reply by ichergui** [reply]
Hey @dat-tran-matternet 

Please check the recipe [here](https://github.com/OE4T/meta-tegra-community/blob/master/recipes-devtools/python/python3-jetson-stats_3.1.2.bb)

I tried it and it is working fine to me 

Let me know if you got any issue.

**Reply by dat-tran-matternet** [reply]
Nice. Thanks for the link
